src/java.desktop/share/classes/javax/swing/plaf/basic/BasicSliderUI.java

```diff
@@ -150,7 +150,10 @@ public class BasicSliderUI extends SliderUI{
 
     /**
      * Constructs a {@code BasicSliderUI}.
+     * @deprecated This constructor was exposed erroneously and will be removed in next version.
+     *             Use {@link #BasicSliderUI(JSlider b)} instead.
```

Comment:
```suggestion
     * @deprecated This constructor was exposed erroneously and will be removed in a future release.
     *             Use {@link #BasicSliderUI(JSlider)} instead.
```

I agree, _“in a future release”_ or _“version”_ would be more accurate.

I verified that it builds with the parameter name, however, it's more common to omit parameter names.